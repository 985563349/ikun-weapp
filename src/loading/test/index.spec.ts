import path from 'node:path';
import simulate from 'miniprogram-simulate';

describe('loading', () => {
  const KLoading = simulate.load(path.resolve(__dirname, '../loading'), 'k-loading', {
    rootPath: path.resolve(__dirname, '../../'),
  });

  test('should render loading and match snapshot', () => {
    const comp = simulate.render(KLoading);
    comp.attach(document.createElement('parent-wrapper'));

    expect(comp.toJSON()).toMatchSnapshot();
  });
});
