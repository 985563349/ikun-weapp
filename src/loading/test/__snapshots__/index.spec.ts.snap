// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading should render loading and match snapshot 1`] = `
<wx-view
  class="k-class k-loading"
  id=""
  style=""
>
  <wx-view
    ariaLabel="loading"
    ariaRole="image"
    class="k-loading__spinner"
  />
</wx-view>
`;
